core/mondoo-azure-inventory.mql.yaml

```
@@ -18,199 +18,199 @@ packs:
     filters: asset.platform == "azure"
     queries:
       - uid: mondoo-asset-inventory-azure-subscription-id
-        title: Retrieve Azure subscription ID
+        title: Azure subscription ID
         docs:
           desc: |
             This query retrieves the Azure subscription id
         mql: azure.subscription.id
       - uid: mondoo-asset-inventory-azure-roleDefinitions
-        title: Retrieve data for all Azure roleDefinitions
+        title: Azure role definitions
         docs:
           desc: |
-            This query retrieves data for all Role definitions in the subscription
+            This query retrieves data for all role definitions in the subscription
         mql: azure.subscription.authorization.roleDefinitions
       - uid: mondoo-asset-inventory-azure-cloudDefender
-        title: Retrieve data for Microsoft Defender for Cloud
+        title: Microsoft Defender for Cloud configuration
         docs:
           desc: |
             This query retrieves data for Microsoft Defender for Cloud
         mql: azure.subscription.cloudDefender
       - uid: mondoo-asset-inventory-azure-storageAccounts
-        title: Retrieve data for all Azure storage accounts
+        title: Azure Storage accounts
         docs:
           desc: |
             This query retrieves data for all storage accounts
         mql: azure.subscription.storage.accounts
       - uid: mondoo-asset-inventory-azure-storageAccounts-containers
-        title: Retrieve data for all containers in Azure storage accounts
+        title: Azure Storage account containers
         docs:
           desc: |
             This query retrieves data for all containers in storage accounts
         mql: azure.subscription.storage.accounts { containers }
       - uid: mondoo-asset-inventory-azure-storageAccounts-blobs
-        title: Retrieve data for all blobs in Azure storage accounts
+        title: Azure storage accounts blobs
         docs:
           desc: |
             This query retrieves data for all blobs in storage accounts
         mql: azure.subscription.storage.accounts { blobProperties }
       - uid: mondoo-asset-inventory-azure-storageAccounts-tables
-        title: Retrieve data for all tables in Azure storage accounts
+        title: Azure storage accounts tables
```

```suggestion
        title: Azure Storage accounts tables
```

You said it's a branded thingy